home.html

```diff
@@ -25,6 +25,8 @@ <h3>Login</h3>
         <div class="col-md-4">
             <h3>Registration</h3>
           If you have not signed up already, please register here.
+            <p>
+                Special characters like # and ; are not allowed.
```

Comment:
Missing a closing tag:
```suggestion
                Special characters like # and ; are not allowed.</p>
```

Reply:
Good catch!